docs/api_requests/patch-block.md

```diff
@@ -19,39 +26,44 @@ curl --unix-socket ${socket} -i \
      -X PUT "http://localhost/drives/scratch" \
      -H "accept: application/json" \
      -H "Content-Type: application/json" \
-     -d "{
-            \"drive_id\": \"scratch\",
-            \"path_on_host\": \"${ro_drive_path}\",
-            \"is_root_device\": false,
-            \"is_read_only\": true
+     -d "{            
+             \"drive_id\": \"scratch\",
+             \"path_on_host\": \"${ro_drive_path}\",            
+             \"is_root_device\": false,            
+             \"is_read_only\": true         
          }"
-
 # Finish configuring and start the microVM. Wait for the guest to boot.
 
-# Resize the block device's backing file and create a filesystem in it.
-truncate --size 100M ${drive_path}
-mkfs.ext4 ${ro_drive_path}
+# Before mounting the block device in the guest:
+# Use another backing file of different size to effectively resize the
+# vm block device.
+touch ${updated_ro_drive_path}
+truncate --size ${new_size}M ${updated_ro_drive_path}
+# Create a filesystem in it.
+mkfs.ext4 ${updated_ro_drive_path}
 
-# Even though the path has not changed, this triggers a device rescan.
+# PATCH the block device to use the new backing file.
 curl --unix-socket ${socket} -i \
      -X PATCH "http://localhost/drives/scratch" \
      -H "accept: application/json" \
      -H "Content-Type: application/json" \
      -d "{
-            \"drive_id\": \"scratch\",
-            \"path_on_host\": \"${ro_drive_path}\"
+             \"drive_id\": \"scratch\",            
+             \"path_on_host\": \"${updated_ro_drive_path}\"         
          }"
+         
+# It's now safe to mount the block device in the guest and use it
+# with the updated backing file.
+```
 
-# Move the backing file.
```

Comment:
Don't we want to still give an example where the backing file is moved? Also looking now would be nice to offer an example where we simulate a hotplug

Reply:
Essentially this is unplug old block file, plug in new block file.

Reply:
Current example is showing how to replace backing file. This is the recommended flow so our example shows exactly that.